drivers/wifi/nrf700x/rpu_fw_patches.ld

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @file
+ * @brief Custom Linker command/script file
+ *
+ * Custom Linker script for the Cortex-M platforms.
+ */
+
+#include <zephyr/linker/sections.h>
+#include <zephyr/devicetree.h>
+
+#include <zephyr/linker/linker-defs.h>
+#include <zephyr/linker/linker-tool.h>
+
+/*
+ * nRF53/52 series ship an external flash that can be used for XIP using QSPI/SPI.
+ *
+ * TODO: In nRF7002 external flash using SPI isn't supported by nrfjprog yet, so,
+ * the code builds but flashing fails.
+ */
+#if CONFIG_BOARD_NRF5340DK_NRF5340_CPUAPP
+#define EXTFLASH_BASE_ADDR 0x10000000
+#define EXTFLASH_SIZE 0x800000
+#elif CONFIG_BOARD_NRF52840DK_NRF52840
+#define EXTFLASH_BASE_ADDR 0x12000000
+#define EXTFLASH_SIZE 0x800000
+#endif
+
+#if USE_PARTITION_MANAGER
+#include <pm_config.h>
+#define EXTFLASH_ADDRESS (EXTFLASH_BASE_ADDR + PM_EXTERNAL_FLASH_ADDRESS)
+#define EXTFLASH_SIZE (PM_EXTERNAL_FLASH_SIZE)
```

Comment:
Doesn't it raise a warning since you redefine the macro without #undef?

Reply:
Well, I hadn't tested with `PARTITION_MANAGED_ENABLED`.